.env

```diff
@@ -2,7 +2,7 @@ VITE_HUB_URL=https://testnet.snapshot.org
 VITE_RELAYER_URL=https://testnet.snapshot.org
 VITE_SCORES_URL=https://score.snapshot.org
 VITE_IPFS_GATEWAY=snapshot.mypinata.cloud
-VITE_DEFAULT_NETWORK=1
+VITE_DEFAULT_NETWORK=5
```

Comment:
@samuveth wdut maybe we can set default network to goerli? we are in testnet anyway

Reply:
For that we will have to change some ENV on fleek and vercel. Now if you open the preview deployment we are on `hub.snapshot.org` but the network is on Goerli.

Reply:
We should handle this in anther PR/Issue

Reply:
https://github.com/snapshot-labs/snapshot/issues/3705